fusequery/query/src/configs/config.rs

```diff
@@ -135,16 +137,68 @@ pub struct Config {
 
     #[structopt(long, env = STORE_API_ADDRESS, default_value = "127.0.0.1:9191")]
     pub store_api_address: String,
+ 
+    #[structopt(long, env = "STORE_API_USERNAME", default_value = "root")]
+    pub store_api_username: User,
 
-    #[structopt(long, env = STORE_API_USERNAME, default_value = "root")]
-    pub store_api_username: String,
+    #[structopt(long, env = "STORE_API_PASSWORD", default_value = "root")]
+    pub store_api_password: Password,
 
-    #[structopt(long, env = STORE_API_PASSWORD, default_value = "root")]
-    pub store_api_password: String,
 
     #[structopt(long, short = "c", env = CONFIG_FILE, default_value = "")]
     pub config_file: String,
 }
+#[derive(Clone, serde::Deserialize, PartialEq, StructOpt, StructOptToml)]
+#[serde(default)]
```

Comment:
Why we need this?

Reply:
I thought this self-defined structure may need the same traits as Config did.

Reply:
Understand.